docs/cloud/managing-airbyte-cloud.md

```diff
@@ -97,76 +106,45 @@ Understanding the following limitations will help you better manage Airbyte Clou
 * Shortest sync schedule: Every 60 min
 * Schedule accuracy: +/- 30 min
 
-## View the sync log summary
-The sync log summary shows if a sync succeeded, failed, cancelled, or is currently running.
+## View the sync summary
+The sync summary displays information about the data moved during a sync.
  
-To view the sync log summary:
+To view the sync summary:
 1. On the [Airbyte Cloud](http://cloud.airbyte.io/) dashboard, click **Connections** in the navigation bar.   
 
     The Connections page displays.
 
 2. Click a connection in the list to view its sync history.
 
-3. Under Sync History, a list of syncs for that connection is displayed.
-
-    The sync status is displayed (Succeeded, Cancelled, Failed, or Running) along with the [summary of the sync log](#sync-log-summary).  
+    Sync History displays the sync status or [reset](https://docs.airbyte.com/operator-guides/reset/) status (Succeeded, Partial Success, Failed, Cancelled, or Running) and the [sync summary](#sync-summary).  
 
-4. For more information about a sync, click the sync log dropdown.  
+    :::note 
+    
+    Airbyte will try to sync your data three times. After a third failure, it will stop attempting to sync.
```

Comment:
@Amruta-Ranade Do you think I should keep or delete this note about Airbyte syncing data three times before stopping?

Reply:
let's keep it for now